plutus-book/doc/authors.adoc

```diff
@@ -24,3 +24,8 @@ She obtained her Master’s degree in Mathematics, also from the University of O
 with the focus of her thesis being the study of abstract computation using
 Category theory, in particular, Turing categories.
 --
+
+[#authors]
```

Comment:
This adds an id to the section. That id is already in use for the section above, so this won't work. This section probably doesn't need an id so I'd just delete it.